packages/controllers/src/snaps/SnapController.ts

```diff
@@ -1780,7 +1766,7 @@ export class SnapController extends BaseController<
    * version.
    * @returns The resulting snap object.
    */
-  async add(args: AddSnapArgs): Promise<PersistedSnap> {
+  private async _add(args: AddSnapArgs): Promise<PersistedSnap> {
```

Comment:
Should we truly make it private?

```suggestion
  async #add(args: AddSnapArgs): Promise<PersistedSnap> {
```

Reply:
While we're at it, I really dislike having `_` in names in general, I think the `SnapController` is the only place we do it. Might as well just make them normal names but have a private modifier.

Reply:
Probably, but that would require more test refactoring than I am strictly willing to tackle right now 😅

Reply:
@ritave Yeah that's a good point, we can address that in a follow-up.

Reply:
Can you make an issue about it?

Reply:
https://github.com/MetaMask/snaps-monorepo/issues/887